web/crux-ui/src/routes.ts

```diff
@@ -531,6 +531,46 @@ class StorageRoutes {
   details = (id: string) => `${this.root}/${id}`
 }
 
+// config bundle
+
+class ConfigBundleApi {
+  private readonly root: string
+
+  constructor(root: string) {
+    this.root = `/api${root}`
+  }
+
+  list = () => this.root
+
+  details = (id: string) => `${this.root}/${id}`
+
+  options = () => `${this.root}/options`
+}
+
+class ConfigBundleRoutes {
+  private readonly root: string
+
+  constructor(root: string) {
+    this.root = `${root}/configbundles`
```

Comment:
I'm thinking on it, which would be more proper this, or with the -, like `config-bundles`